policyengine_us/parameters/gov/states/ny/tax/income/deductions/itemized/phase_out/start.yaml

```diff
@@ -1,18 +1,25 @@
 description: Federal AGI over this threshold phases out itemized deductions for NY State income tax.
 SINGLE:
   2021-01-01: 282_400
+  2023-01-01: 313_200
 JOINT:
-  2021-01-01: 310_600
+  2021-01-01: 338_850
+  2023-01-01: 375_850
 SURVIVING_SPOUSE:
-  2021-01-01: 310_600
+  2021-01-01: 338_850
+  2023-01-01: 375_850
 SEPARATE:
   2021-01-01: 169_400
+  2023-01-01: 187_900
```

Comment:
we can actually uprate this parameter value:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/4686ea32-9174-49ee-b332-b28fca30ae13)